storagenode/orders/sender.go

```diff
@@ -222,33 +228,47 @@ func (sender *Sender) settle(ctx context.Context, log *zap.Logger, satelliteID s
 		return OrderError.New("failed to start settlement: %v", err)
 	}
 
+	var errList errs.Group
+	errHandle := func(logErrMsg string, err error, fields ...zap.Field) {
+		log.Error(logErrMsg, append([]zap.Field{zap.Error(err)}, fields...)...)
```

Comment:
nit - I'm possibly wrong but it doesn't seem like much more lines to just have the `log.Err` calls and `errList.Add`s in the err check instead of using the `errHandle` function, but this is fine too

Reply:
I agree. I conserved because it already existed and it also had a couple of lines.

I'm fine to remove it and directly do what each call to this should do; it's clearer to me than having this handler `errHandle` function

Reply:
By the way, this suggestion was applied.

Reply:
thanks for addressing!